climateconnect/climateconnect
# Feat/event project cards

This PR introduces the event project cards necessary for the event calendar feature (https://github.com/climateconnect/climateconnect/issues/1189). Before this PR is reviewed and merged, #1250 should be reviewed and merged.

## Concrete changes
- Add yellow indication of start date and time for event types on the project card
- Make indication grey for events that are over
- Show start and end date in indication for multi-day-events
- Show project type with corresponding icon on project cards

## To test 
- Share one idea type project, one project type project
- Share at least three event type projects: One which has its end_date in the past, one which spans multiple days and one which spans only one day and has its end_date in the future.
- Check whether the date incidations work as described
- Check whether project type incidations work as described
- Check if this also works in hubs

## Screenshots
![Screenshot from 2023-07-06 17-51-45](https://github.com/climateconnect/climateconnect/assets/10531079/1cd0047e-106d-47d7-a753-4345f12d8ebc)
